### PR TITLE
Add 19892

### PR DESCRIPTION
**Pull request by DariaElagina**

### CODE REVIEW

**Review by Hasnake — APPROVED** [review]

**Review by Hasnake — APPROVED** [review]

File: LondonBritishLibrary/add/BLadd19892.xml [hunk 1]
```diff
@@ -0,0 +1,272 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"
+?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+   schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" type="mss" xml:id="BLadd19892" xml:lang="en">
+    <teiHeader xml:base="https://betamasaheft.eu/">
+        <fileDesc>
+            <titleStmt>
+                <title>Malkǝʾa Māryām, qǝddase Māryām, maṣḥafa saʿātāt, saʿātāt</title>
```
Comment (DenisNosnitsin1970) [inline LondonBritishLibrary/add/BLadd19892.xml:8]: Please correct: Qǝddase, Maṣḥafa, Saʿātāt

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

File: LondonBritishLibrary/add/BLadd19892.xml (hunk 1 above)
Comment (Getinet309) [reply LondonBritishLibrary/add/BLadd19892.xml:8]: <colophon xml:id="coloph1" xml:lang="gez">
  <locus target="#201r"></locus>
  ዛቲ፡ ጸሎት፡ ዘ<persName ref="PRS13556HabtaM" role="scribe">አባ፡ ሐብተ፡ ማርያም፡</persName>
  <sic>ዘደብረ፡</sic> ዘጸሐፍክዋ፡ በሮምያ፡
  ውስተ፡ <placeName ref="INS0302SanStefano">ደብረ፡ ቅዱስ፡ እስጢፋኖስ፡ ዘማኅበረ፡ ኢትዮጵያውያን፡</placeName>
  <sic>በመዋዕለ፡</sic> <persName ref="PRS9682UrbanVI">ሊቀ፡ ጳጳሳት፡ <sic>ኡባኖስ።</sic></persName>
  ሳምናይ፡ ዘመንበረ፡ ቅዱስ፡ ጴጥሮስ።
  <date>ወዓመተ፡ ምሕረትሂ፡ ፲፻፮፻ወ፵፡ እምልደተ፡ ክርስቶስ፡ እግዚእነ።</date>
  <gap reason="ellipsis"></gap>

**Review by Getinet309 — CHANGES_REQUESTED** [review]
I hope there is missed text next to  "በደብረ"

**Comment by DariaElagina** [comment]
Dear Getinet, 
thank you for your correction. Actually, the text of the manuscript is imperfect and reads በዋዕለ፡ and has nothing after በደብረ. That is also why those words are surrounded by a sic element, which indicates that the text follows the manuscript, although the text appears to be incorrect.

**Pull request merged**